liquidity-rewards/contracts/UniswapPoolRewards.sol

```diff
@@ -1,5 +1,138 @@
 // SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
 
-contract UniswapPoolRewards {
-  
+import "./dependencies/PoolTokenWrapper.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/math/Math.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+
+contract UniswapPoolRewards is PoolTokenWrapper, Ownable {
```

Comment:
Does `Ownable` allow for _multiple_ owners, or only one?

Reply:
It allows only one owner. if you want more details, this is the comment from the library:
```
/**
 * @dev Contract module which provides a basic access control mechanism, where
 * there is an account (an owner) that can be granted exclusive access to
 * specific functions.
 *
 * By default, the owner account will be the one that deploys the contract. This
 * can later be changed with {transferOwnership}.
 *
 * This module is used through inheritance. It will make available the modifier
 * `onlyOwner`, which can be applied to your functions to restrict their use to
 * the owner.
 */
```